presto-main/src/main/java/io/prestosql/operator/HttpPageBufferClient.java

```diff
@@ -174,7 +180,10 @@ public HttpPageBufferClient(
             Ticker ticker,
             Executor pageBufferClientCallbackExecutor,
             boolean isSnapshotEnabled,
-            QuerySnapshotManager querySnapshotManager)
+            QuerySnapshotManager querySnapshotManager,
+            FailureDetector failureDetector,
+            boolean timeoutFailureDetection,
```

Comment:
variable name doesnt clearly tells is boolean type. can use something like detectTimeoutFailures

Reply:
fixed